tests/core/models.py

```diff
@@ -5,10 +5,29 @@
 from django.db import models
 
 
+class AuthorManager(models.Manager):
```

Comment:
I suggest removing from here and defining solely in tests and documentation (please see [main comment](https://github.com/django-import-export/django-import-export/pull/1371#issuecomment-1001748740)).

Reply:
That makes sense to me, I noticed after I finished setting all that up that most of the tests define their own versions of resources and such.

Reply:
So this might just be my issue but I don't see a lot of clean ways to do this. I couldn't find any clear cut way to dynamically change a model's manager only for a test case, and I didn't have any luck creating models within a test case (which makes sense, they're not in the apps).  

What's confusing to me on that front is that I see another test that successfully makes models (BookWithChapters from the test_resources module), but I can't for the life of me tell how they do it -- I get a 'no such table' error). 

Do you have any thoughts on the right way to do this?